src/Controller/UI/MdAppController.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+namespace App\Controller\UI;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+
+use App\Form\MdappType;
+
+use Doctrine\ORM\Query;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+use App\Entity\Dataset;
+use App\Entity\DatasetSubmission;
+use App\Handler\EntityHandler;
+use App\Util\MdappLogger;
+
+/**
+ * The MDApp controller.
+ */
+class MdAppController extends AbstractController
+{
+    /**
+     * Jira issue tracker.
+     *
+     * @var string issueTrackingBaseUrl
+     */
+    protected $issueTrackingBaseUrl;
+
+    /**
+     * Class constructor.
+     *
+     * @param EntityHandler $entityHandler         A Pelagos EntityHandler instance.
+     * @param string        $issueTrackingBaseUrl  Issue tracker base URL.
+     */
+    public function __construct(EntityHandler $entityHandler, string $issueTrackingBaseUrl)
+    {
+        $this->issueTrackingBaseUrl = $issueTrackingBaseUrl;
+        $this->entityHandler = $entityHandler;
```

Comment:
Global variable in the class missing for entityHandler

Reply:
added var